remme/shared/forms/transaction.py

```diff
@@ -61,3 +61,27 @@ def validate_reverse(form, field):
 
         if field.data != 'false':
             raise validators.StopValidation('Incorrect reverse identifier.')
+
+
+class ListReceiptsForm(ProtoForm):
+
+    ids = fields.StringField()
+
+    if isinstance(ids, list):
+        ids = fields.FieldList(ids, min_entries=1)
```

Comment:
```suggestion
        ids = fields.FieldList(fields.StringField(), min_entries=1)
```

Reply:
@anastasiia-bilova, [FieldList](https://github.com/Remmeauth/remme-core/pull/298/files#r271793641)

Reply:
@Alladin9393, indicate `fields.StringField()` to make it clear, `ids` it is not clear

Reply:
@anastasiia-bilova, done.